sklearn/preprocessing/tests/test_data.py

```diff
@@ -1334,13 +1334,13 @@ def test_quantile_transform_subsampling():
     ROUND = 5
     inf_norm_arr = []
     for random_state in range(ROUND):
-        transformer = QuantileTransformer(
+        transformer_s0 = QuantileTransformer(
```

Comment:
Can we undo this renaming again now that there is a separate test?

Reply:
Yes, my bad. I think something went wrong in one of the rebases.